src/lookup/docs/demo.js

```diff
@@ -55,6 +50,25 @@ angular.module('dd.ui.demo').controller('LookupDemoCtrl', ['$scope', '$http', '$
                 { name: 'Provider item 2' }
             ];
     };
+    
+    $scope.getGroupedData = function(query) {
```

Comment:
$scope.getGroupedData return flat array, consider changing this method name to $scope.getData